public/src/components/channelManagement/bannerTests/bannerTemplateSelector.tsx

```diff
@@ -1,74 +0,0 @@
-import React from 'react';
```

Comment:
This has been renamed to `BannerUiSelector` bit GH is showing as a deletion and creation.